src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermDays = 7;
+  const shortTermDays = 3;
+  const carCostPerDay = 40;
+  const mediumRentTermDiscount = 20;
+  const longRentTermDiscount = 50;
```

Comment:
Here it is better to use contacts:

![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/54860641/13bead3f-bab3-42f9-9a66-34b89939b4fa)